src/pages/RFOX/components/Claim/types.ts

```diff
@@ -17,8 +17,3 @@ export type RfoxClaimQuote = {
   stakingAmountCryptoBaseUnit: string
   index: number
 }
-
-export enum ClaimStatus {
-  Available = 'Available',
-  CoolingDown = 'Cooling down',
```

Comment:
Oh, I see you actually changed _from_ the suggestion above. Any reason why? 
If we are going to keep the change, maybe "Unavailable"?

Reply:
Yeah, wanted to keep this general since "cooling down" is very rFOX specific. Unavailable works, since it'll not be user facing since ideally we'll use a translation.